### ISSUE TITLE
🐛 Pre-Release version does not work with Biome v2 or Biome v1

### ISSUE THREAD

**Issue by sinomsinom** [biomejs/biome-vscode#546]
### VS Code version

1.98.2

### Extension version

2025.3.151143

### Biome version

2.0.0-beta.1 (none displayed in the statusbar)

### Operating system

- [x] Windows
- [ ] macOS
- [ ] Linux

### Description

When installing Biome 2.0.0-beta.1 (or Biome 1.9.4) and the pre-release version of the extension as recommended by the Biome 2 beta blog post, the extension does not work correctly. 
- It can't be registered as a formatter and does not show any linter errors. 
- When trying to use the "Biome: Download CLI" option it does not show 2.0.0-beta.1 as an option. Trying to select any other version throws the error "Cannot find module './index-CJAZtKJE.js'. 
- The icon in the bottom right shows a check-mark but does not show a version and there is no popup asking to install a biome version either.
These issues also persisted when switching back to Biome 1.9.4, or doing a reinstall with 1.9.4 and using the pre-release extension. 
The non pre-release extension seems to work fine with Biome 1.9.4

### Steps to reproduce

1. Add Biome 2.0.0-beta.1 to project (Or Biome 1.9.4)
2. Install Biome extension in pre-release mode
3. Try using the extension

### Expected behavior

The extension should be able to be registered as a formatter and should show linter error. 
The Biome: Download CLI option should not throw an error when used.
Indicator on bottom right should show a biome version

### Does this issue occur when using the CLI directly?

No

### Link to a minimal reproduction
https://github.com/sinomsinom/biome-repro-1743157065790
This issue is in any repo I tried (including an empty repo created with the npm create @biomejs/biome-reproduction command) so a repro repo doesn't seem useful here

### Logs

```plain text
Output:
2025-03-28 10:35:26.190 [info] 





2025-03-28 10:35:26.190 [info] Biome extension 2025.3.151143 activated
2025-03-28 10:35:26.190 [info] Biome extension started
2025-03-28 10:35:26.190 [info] User-facing commands registered
2025-03-28 10:35:26.190 [info] Started listening for lockfile changes
2025-03-28 10:35:26.190 [info] Started listening for configuration changes
2025-03-28 10:35:26.190 [info] Started listening for active text editor changes



biome rage --daemon-logs:
CLI:
  Version:                      2.0.0-beta.1
  Color support:                true

Platform:
  CPU Architecture:             x86_64
  OS:                           windows

Environment:
  BIOME_LOG_PATH:               unset
  BIOME_LOG_PREFIX_NAME:        unset
  BIOME_CONFIG_PATH:            unset
  NO_COLOR:                     unset
  TERM:                         unset
  JS_RUNTIME_VERSION:           v22.14.0
  JS_RUNTIME_NAME:              node
  NODE_PACKAGE_MANAGER:         unset

Biome Configuration:
  Status:                       Loaded successfully
  Path:                         C:\Users\<path-to-project>\biome.json
  Formatter enabled:            true
  Linter enabled:               true
  Assist enabled:               true
  VCS enabled:                  false

Workspace:
  Open Documents:               0

Discovering running Biome servers...
```

**Comment by sinomsinom** [comment]
After looking through the closed issues this seems to be the same issue as https://github.com/biomejs/biome-vscode/issues/533 however that issue was closed by a bot without actually being fixed. The issue still exists.

**Comment by ematipico** [comment]
> so a repro repo doesn't seem useful here

**It is useful**, because you might have some user settings that other people don't. So please, create one and share it with us. For example, I just tried your exact setup locally and it works as expected. 

The previous issue was closed because *noone* provided a reproduction for us.

**Comment by sinomsinom** [comment]
@ematipico https://github.com/sinomsinom/biome-repro-1743157065790

**Comment by sinomsinom** [comment]
About vsc user settings. 
This issue also happens with a completely empty settings.json file.

I have also tried:
- uninstalling the extension and only installing the pre-release extension (in case switching from release to pre-release messes anything up)
- uninstalling all biome versions and only installing 1.9.4 (though afaik it should only use the one in the project anyways not any global ones)

**Comment by sinomsinom** [comment]
After doing a git bisect the specific commit to have broken this seems to be https://github.com/biomejs/biome-vscode/commit/602c510201c59d072315769728050c94152889c3

**Comment by lgarron** [comment]
I'm also having issue trying out Biome beta (`2.0.0-beta.1`) with the pre-release extension (2025.3.151143).

The CLI works fine on the commandline, but running "Format document" seems to do nothing and I can't find errors/output/indicators about what happened. The extension logs are completely empty files.

**Comment by nhedger** [comment]
I cannot reproduce this issue with Biome `1.9.4`.

However, when using `2.0.0-beta.1`, the extension fails to start an LSP session because the provided configuration file has errors. Specifically, the `organizeImports` key is no longer supported. 

```diff
{
	"$schema": "./node_modules/@biomejs/biome/configuration_schema.json",
+	"organizeImports": {
+		"enabled": true
+	},
	"linter": {
		"enabled": true,
		"rules": {
			"recommended": true,
			"suspicious": {
				"noDoubleEquals": "error"
			}
		}
	}
}
```


This causes the startup to fail (and a warning is displayed to that effect).

![Image](https://github.com/user-attachments/assets/2dd9ef3f-f8ce-4581-a64b-583e9b621a65)

To fix the configuration, use the new `assist` key, and reload your editor.

```diff
{
	"$schema": "./node_modules/@biomejs/biome/configuration_schema.json",
+	"assist": {
+		"enabled": true,
+		"actions": {
+			"source": {
+				"organizeImports": "on"
+			}
+		}
+	},
	"linter": {
		"enabled": true,
		"rules": {
			"recommended": true,
			"suspicious": {
				"noDoubleEquals": "error"
			}
		}
	}
}
```

**Comment by sinomsinom** [comment]
This config issue probably has nothing to do with this issue. 

I don't get any error whatsoever and I made sure the config would be correct. Using the command line interface I can use biome 1 and 2 no problem with the corresponding configs. It only happens if I try to use the plugin with a version newer than https://github.com/biomejs/biome-vscode/commit/602c510201c59d072315769728050c94152889c3 which makes me think it might be an issue where the extension just can't find the workspace or any files in it at all for some reason? But I also can't find any diagnostic messages, errors, or anything that would say as much.

With any commit before that specific one the biome icon in the ribbon at the bottom also correctly shows the used biome version, however with that commit and newer ones it does not while still showing the "up and running" on hover. This makes me think it might also not be able to find the biome LSP (or potentially just unable to find biome in general), but again there is no error message to indicate this. The extension also doesn't even report parsing errors. It basically just doesn't do anything while in that state.

The only error message I get is when trying to download a biome version through the "Biome: Download CLI" option which throws a "Cannot find module './index-CJAZtKJE.js'" error. This is also not an issue with versions older than https://github.com/biomejs/biome-vscode/commit/602c510201c59d072315769728050c94152889c3

Here's an image of that error message:
![Image](https://github.com/user-attachments/assets/7b434dcc-fb61-4e2c-adda-a8549c7b8bb1)

**Comment by nhedger** [comment]
Thanks for the details!

**Comment by nhedger** [comment]
Sorry, but I'm unable to reproduce it.

Works as expected with 1.9.4, and fails as expected using 2.0.0-beta.1 because of the configuration issue.

I'm trying to reproduce on macOS, so maybe it's specific to Windows, I'll boot a VM to see if I can reproduce on Windows.

**Comment by nhedger** [comment]
Tried it on Windows 11, and the results are the same. I cannot reproduce the issue.

Could you try disabling all other extension to see if one might be causing an issue with the Biome extension ?

**Comment by sinomsinom** [comment]
I just tried disabling every other extension I have installed and the bug still happened. I then also tried to reinstall the biome extension with everything else enabled and it still happened. 

As I posted here before using a completely empty settings.json to start with and then re-installing the extension also did not fix the issue.

**Comment by nhedger** [comment]
Thanks for looking.

I spoke too fast in my previous message. I wasn't running on the pre-release version, but after reading your message, I tried again and was able to reproduce the issue this time.


I found what is causing the issue, and it is specific to Windows. I'll come up with a fix as soon as possible and release a new pre-release version shortly after.

**Comment by nhedger** [comment]
@sinomsinom Would you mind trying the following VSIX to check if the fix works for you?

[biome.vsix.zip](https://github.com/user-attachments/files/19922343/biome.vsix.zip)

**Comment by sinomsinom** [comment]
Sorry for the late reply. Yes this version seems to have fixed the issue!

**Comment by nhedger** [comment]
Thanks for confirming. I'll merge and release a fresh version
